### DISCUSSION TITLE
New 1.6 version of PCB

### DISCUSSION THREAD

**Discussion by AndySmithers** [chickey/RS485-WiFi-EPEver#27] [General]
Hi Colin,
On the new version 1.6 of the PCB (purchased from Lectronz), you seem to have ditched the 3.3V regulator and now route VCC from the Modbus cable through a diode to +5V on the WEMOS Mini. My EPEVER 1206N provided only +5V on the 485 connector so the voltage at the WEMOS was only ~4.4V. I was getting constant random resets.  I removed the series diode and now the random restarts have "almost" gone away. I still see a reset once every few minutes though. 
I'm going to try to track down the reason for these occasional resets but wonder if you have any ideas.
Also, I want to try to strip out all the database and MQTT stuff for a simple, basic web-only interface. Any pointers gratefully received on this.

**Comment by chickey** [comment]
When you removed the Diode did you bridge the solder joint so still only powering it from the modbus cable?  I would recommend you charge it only with one source.

I dropped the 3.3v regulator onboard as there is already one on the wemos and in the early days i had issues with certain regulators so it seemed redundant to have two things onboard doing the same job.  Re your problems i do wonder if it is still power based

Re modifying the codebase there is no simple tips, would be a case of removing everything which isn't required.  I would instead recommend esphome if your using home assistant, you can configure it so that it only queries the registers you want to read and give you a simple web interface.  I've not looked into using it without homeassistant but it may be possible.  For ESPHome you'll need to make a small mod to the board though and join the pins where D1 and D2 plugin.  I would do this on the carrier board and not the wemos D1 mini direct.

You may be better off discussing in the small discord we have setup
https://discord.gg/hrwt6pSv

**Reply by AndySmithers** [reply]
Hi Colin, thanks for quick reply.
Yes, I should have said "bridged" rather than "removed" the diode.
I too think the problems are still power related.
I note that you specifically state that this board hasn't been tested with the new negative ground units - which is what mine is. But I can't think of any reason why that would make a difference.
I'll take a look at the ESPHome usage model, thanks for the idea...

**Comment by adrianheff** [comment]
[logs_epever_run.txt](https://github.com/chickey/RS485-WiFi-EPEver/files/13943376/logs_epever_run.txt)
Hi Colin 
I purchased an adapter from here recently 
I am trying to get data from my epever charger to home assistant.
When i use the epever.yaml code and upload it to my di mini device I get the error as per attached log file, 'charger_status' was not declared in this scope ?
Any idea why this is happening
